application/testing/CMakeLists.txt

```diff
@@ -750,8 +750,10 @@ if(F3D_PLUGIN_BUILD_HDF)
   f3d_test(NAME TestNoRenderAnimation DATA small.ex2 ARGS  --load-plugins=hdf --animation-time=0.003 REGEXP "-994.473, 33.9259" NO_RENDER)
 
   # Test animation time clamping
-  f3d_test(NAME TestAnimationTimeLimitsHigh DATA small.ex2 ARGS ARGS --load-plugins=hdf --animation-time=10)
-  f3d_test(NAME TestAnimationTimeLimitsLow DATA small.ex2 ARGS ARGS --load-plugins=hdf --animation-time=-5)
+  f3d_test(NAME TestAnimationTimeLimitsHigh DATA small.ex2 ARGS ARGS --load-plugins=hdf --animation-time=10 REGEXP "Animation time 10 is outside of range")
+  f3d_test(NAME TestAnimationTimeLimitsLow DATA small.ex2 ARGS ARGS --load-plugins=hdf --animation-time=-5 REGEXP "Animation time -5 is outside of range")
```

Comment:
thats not right, there tests are baseline tests, by adding a REGEXP, the baseline is not tested anymore.

Reply:
oh ok I thought I could do both at once. I'll split them. Unless the warnings are already tested somewhere else I missed?

Reply:
BTW its a ctest limitation